var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -126,7 +126,8 @@ class QtBase(QtPackage):
     with when("platform=linux"):
         depends_on("libdrm")
 
-    depends_on("at-spi2-core", when="+accessibility")
+    with when("platform=linux"):
+        depends_on("at-spi2-core", when="+accessibility")
```

Comment:
It'd be great if we could merge these two `when("platform=linux")` blocks to reduce duplicated code